git-hg/updatetags8.sh

```diff
@@ -0,0 +1,134 @@
+#!/bin/bash
+#
+# ===========================================================================
+# (c) Copyright IBM Corp. 2018 All Rights Reserved
+# ===========================================================================
+#
+# This code is free software; you can redistribute it and/or modify it
+# under the terms of the GNU General Public License version 2 only, as
+# published by the Free Software Foundation.
+#
+# This code is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+# version 2 for more details (a copy is included in the LICENSE file that
+# accompanied this code).
+#
+# You should have received a copy of the GNU General Public License version
+# 2 along with this work; if not, see <http://www.gnu.org/licenses/>.
+#
+# ===========================================================================
+
+if [ $# -lt 2 ]; then
+   echo Usage: $0 '[jdk8u|jdk9] TAGS'
```

Comment:
Note to self - we should *really* move jdk9 to jdk9u now.  Let's do this the moment we accept this request and start building 9 binaries again.

Reply:
Have adjusted the script to handle `jdk9*` so it should be ok when we switch